taskcluster/async/scheduler.py

```diff
@@ -1,293 +0,0 @@
-# coding=utf-8
```

Comment:
And, it looks like, remove scheduler support?  That should probably be broken out into a separate commit so it can be seen in the git log.